code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -226,6 +226,11 @@ datum/reagent/medicine/adminordrazine/on_mob_life(var/mob/living/carbon/M as mob
 	..()
 	return
 
+datum/reagent/medicine/adminordrazine/nanites
```

Comment:
Just change it so it dosent remove this lot and I am fine this is the real issue I have:
	M.hallucination = 0
M.setBrainLoss(0)
M.disabilities = 0
M.eye_blurry = 0
M.eye_blind = 0
M.SetWeakened(0)
M.SetStunned(0)
M.SetParalysis(0)
M.silent = 0
M.dizziness = 0
M.drowsyness = 0
M.stuttering = 0
M.confused = 0
M.sleeping = 0
M.jitteriness = 0